add a missing initialization that causes a crash when parsing invalid volume restoration tables (Problem identified by Luigi Auriemma, re #67)

From: lennart <lennart@fefdeb5f-60dc-0310-8127-8f9354f1896f>

git-svn-id: svn://svn.0pointer.net/pulseaudio/trunk@1451 fefdeb5f-60dc-0310-8127-8f9354f1896f
---

 src/modules/module-volume-restore.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/modules/module-volume-restore.c b/src/modules/module-volume-restore.c
index efa59f4..97d0d5e 100644
--- a/src/modules/module-volume-restore.c
+++ b/src/modules/module-volume-restore.c
@@ -433,7 +433,8 @@ int pa__init(pa_core *c, pa_module*m) {
     u->subscription = NULL;
     u->table_file = pa_xstrdup(pa_modargs_get_value(ma, "table", NULL));
     u->modified = 0;
-    
+    u->sink_input_hook_slot = u->source_output_hook_slot = NULL;
+
     m->userdata = u;
     
     if (load_rules(u) < 0)
